runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -3891,7 +3891,8 @@ void mcmReleaseOneNode(c_nodeid_t node, struct perTxCtxInfo_t* tcip,
   DBG_PRINTF(DBG_ORDER,
              "dummy GET from %d for %s ordering",
              (int) node, dbgOrderStr);
-  uint64_t flags = (mcmMode == mcmm_msgOrdFence) ? FI_FENCE : 0;
+  uint64_t flags = (mcmMode == mcmm_msgOrdFence) ?
+                      FI_FENCE | FI_DELIVERY_COMPLETE : 0;
```

Comment:
for clarity sakes, can you put `FI_FENCE | FI_DELIVERY_COMPLETE` in parentheses?